kubectl-openebs/cli/command/describe/pvc_info.go

```diff
@@ -194,11 +194,15 @@ func RunPVCInfo(cmd *cobra.Command, pvcs []string, ns string, openebsNs string)
 
 				// fetching the underlying TargetPod for the PV, to display its relevant details and notify the user
 				// if the TargetPod is not found.
-				targetPod, err := clientset.GetCstorVolumeTargetPod(item.Name, item.Spec.VolumeName)
+				targetPod, err := clientset.GetCVTargetPod(item.Name, item.Spec.VolumeName)
```

Comment:
A variable namechange request, as go experts suggest using small variable names to indicate small some variable being used for the short term. Moreover the name target comes twice in the same line.

```suggestion
				tgtPod, err := clientset.GetCVTargetPod(item.Name, item.Spec.VolumeName)
```